srcpkgs/avldrums.lv2/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'avldrums.lv2'
+pkgname=avldrums.lv2
+version=0.4.1
+revision=1
+build_style=gnu-makefile
+makedepends="lv2 libglib-devel pango-devel cairo-devel mesa glu-devel pkg-config"
```

Comment:
Should be `mesa-devel`, not mesa here. 

`pkg-config` should go in `hostmakedepends`